### ISSUE TITLE
Is it possible to have Wisdom documentation available with one page per chapter ?

### ISSUE THREAD

**Issue by Riduidel** [wisdom-framework/wisdom#152]
Having the full documentation as one page may be useful (specifically for printing), but it would also be nice to have it splitted with one page per chapter (JBoss projects tend to do that ...)

**Comment by remi-parain** [comment]
As the doc subject start to pollute some google groups threads maybe this thread should be updated ?

**Comment by cescoffier** [comment]
Yes, if anyone can have a look on how to to that using Ascidoctor, it would
be awesome.

2015-03-24 12:31 GMT+01:00 PARAIN Rémi <notifications@github.com>:

**Comment by hassanchris** [comment]
Excellent
